### PR TITLE
use new ascii printing functionality for flow tables in katnetic

### PR DESCRIPTION
**Pull request by smolkaj**

### CODE REVIEW

**Comment by arjunguha** [comment]
Oh, I see. `string_of_flowTable` is not defined in terms of `format_flowTable`. This is the convention we've had so far, but I don't know if OCaml printers make sense what the new `string_of_flowTable` does.

@jcollard why don't you lookup the Format module:

https://ocaml.org/learn/tutorials/format.html

Does it make sense for the new printer to use it? If so, do so. Otherwise, just merge this.

File: frenetic/katnetic.ml [hunk 1]
```diff
@@ -43,14 +43,14 @@ module Global = struct
...
+        (fun sw -> Optimize.specialize_policy sw local_pol
+                    |> NetKAT_LocalCompiler.compile
+                    |> NetKAT_LocalCompiler.to_table sw
+                    |> (fun t -> (sw, t)))
         switches in
     let print_table (sw, t) =
-      Format.fprintf fmt "[global] Flowtable for Switch %Ld:@\n@[%a@]@\n@\n"
-        sw
-        SDN_Types.format_flowTable t in
+      Format.fprintf fmt "@[%s@]@\n@\n"
```
Comment (arjunguha) [inline frenetic/katnetic.ml]: I think this `fmt` is a free variable.

Comment (smolkaj) [inline frenetic/katnetic.ml]: It's actually declared a few lines above.

**Comment by smolkaj** [comment]
I think it's fine. I restarted Travis.

**Comment by arjunguha** [comment]
Blah, still fails. But hey, this is a simple fix. Just commit it straight to master, IMO.

**Pull request merged**